runtime/runtime/config/config.go

```diff
@@ -94,6 +112,14 @@ type SQLServer struct {
 	// Valid formats are "hostname", "hostname:port", and "/path/to/unix.socket".
 	Host string `json:"host"`
 
+	// MinConnections is the minimum number of open connections to use
+	// for this database. If zero it defaults to 2.
+	MinConnections int `json:"min_connections"`
+
+	// MaxConnections is the maximum number of open connections to use
+	// for this database. If zero it defaults to 30.
+	MaxConnections int `json:"max_connections"`
+
```

Comment:
Was this moved by accident? They belong on the database struct

Reply:
I had a merge conflict this morning when rebasing against main and it looked like they had been moved into the server struct.

Will move them back